ext/reflection/php_reflection.c

```diff
@@ -5921,8 +5921,7 @@ ZEND_METHOD(ReflectionProperty, isVirtual)
 	_property_check_flag(INTERNAL_FUNCTION_PARAM_PASSTHRU, ZEND_ACC_VIRTUAL);
 }
 
-/* {{{ Returns whether this property is default (declared at compilation time). */
-ZEND_METHOD(ReflectionProperty, isDefault)
+static void _property_check_dynamic(INTERNAL_FUNCTION_PARAMETERS, bool dynamicTrue) /* {{{ */
```

Comment:
nit: feel free to omit the `{{{` and `/* }}} */` tags

Reply:
This parameter name is more in line with the conventions:

```suggestion
static void _property_check_dynamic(INTERNAL_FUNCTION_PARAMETERS, bool is_dynamic) /* {{{ */
```

Reply:
I think that might get confusing since I use `isDynamic` to hold the result of checking if the property is dynamic a few lines lower down - how about `dynamic_true`?

Reply:
Hm, ok, it's ok as long as you consistently use the camel_case style ^^ At least, most code in php-src uses this convention so the camelCase looks a bit weird for me here.

Reply:
Isn't camelCase with the capitals, and snake_case what is used?

Reply:
> Isn't camelCase with the capitals, and snake_case what is used?

Sorry, I'm not in my sharpest form today

Reply:
And again, thank you, I support this feature very much, the original method name is very misleading. I'd say that the change is pretty much self contained, but RMs should approve it nevertheless. Please also write an upgrading note once you have the green light.